### ISSUE TITLE
[BUG] CKEditor isn't removed from the DOM

### ISSUE THREAD

**Issue by kuhelbeher** [artf/grapesjs-plugin-ckeditor#37]
CKEditor isn't removed from the DOM after component deselected. It could cause weird bugs like double CKEditor:

![image](https://user-images.githubusercontent.com/37115006/82315547-1d2fa680-99d4-11ea-93f4-c62087499974.png)

![image](https://user-images.githubusercontent.com/37115006/82315496-0d17c700-99d4-11ea-89e7-a15f2943d8b8.png)

**Comment by Houst0n345** [comment]
Have the same bug. After init Ckeditor toolbar and draggin component to another place we have 2 toolbars. 
Resolve with help of this command:
`editor.on('sorter:drag:end', ({ srcEl }) => {
      if (srcEl) {
        $(document).find(`#cke_${srcEl.id}`).remove();
      }
    });`

**Comment by ronaldohoch** [comment]
@[jackyhuynh0407](https://github.com/jackyhuynh0407) will you open a PR with it?

**Comment by jackyhuynh0407** [comment]
Yes I have other solution. But i forgot to update it. I'm not open PR yet

```js
      let rteToolbar = editor.RichTextEditor.getToolbarEl();
      [].forEach.call(rteToolbar.children, (child) => {
        if(child.id == `cke_${el.id}`) child.remove()
      	else child.style.display = 'none';
      });
```

**Comment by ronaldohoch** [comment]
@jackyhuynh0407 nice, thank you!
@artf update the plugin to us 🙈
